packages/react-kit/src/components/Selectbox/Selectbox.tsx

```diff
@@ -42,14 +42,14 @@ export type TFullSelectboxProps = TControlProps<ReactText | undefined> & {
 	Anchor: ComponentClass<TSelectboxAnchorProps>;
 	Menu: ComponentType<TMenuProps>;
 	Popover: ComponentType<TPopoverProps>;
-
+	isOpened?: boolean;
+	onToggle?: (isOpened: boolean) => void;
```

Comment:
Let's replace this with `TControlProps<boolean | undefined, 'isOpened', 'onToggle'>` at the top of this props type declaration